questions/questions/_utilities.py

```diff
@@ -26,6 +27,12 @@ def random_answer_options_pitch(len_of_list=4, correct_answer=None):
     random.shuffle(random_answer_list)
     return random_answer_list
 
+def random_root_position_major_triad():
```

Comment:
I wonder if this belongs as a method on the question class since this seems tightly coupled to the question itself.  I'm open to have my mind changed on this if we think there will be other scenarios where we want a random root position major triads for other questions.  Seems like this is probably pretty isolated to this question, though.

Reply:
Great question! my thought was that getting a root position major triad at random could conceivably be useful to a plethora of questions which is why I put it there so that it could be easily called on later for other contexts.